proxy/src/telemetry/metrics/labels.rs

```diff
@@ -88,6 +125,153 @@ pub struct DstLabels {
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 pub struct TlsStatus(ctx::transport::TlsStatus);
 
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+pub enum TlsConfigLabels {
+    Reloaded,
+    InvalidTrustAnchors,
+    InvalidPrivateKey,
+    InvalidEndEntityCert,
+    Io { path: PathBuf, errno: Option<Errno>, },
+}
+
+mk_err_enum! {
```

Comment:
I think all this errno stuff should be its own module. In fact it duplicates functionality in the `libc` crate so I'm not sure how necessary it is. It would be good to clean this up real soon.